package.json

```diff
@@ -99,9 +99,9 @@
     "@storybook/addons": "^5.2.4",
     "@storybook/react": "^5.2.4",
     "@storybook/theming": "^5.2.4",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^8.0.3",
+    "@testing-library/jest-dom": "4.2.4",
+    "@testing-library/react": "9.3.2",
+    "@testing-library/user-event": "8.1.0",
```

Comment:
The trick to fix the tests was to update `@testing-library/user-event`
I suggest we use strict version numbers for each of these so that we can ensure that both our local environment and Circle CI use the same package versions for tests.